pkg/timeutils/utils_test.go

```diff
@@ -9,22 +9,38 @@ import (
 func TestGetTimestamp(t *testing.T) {
 	now := time.Now()
 	cases := []struct{ in, expected string }{
-		{"0", "-62167305600"}, // 0 gets parsed year 0
```

Comment:
Reviewers - make sure you're ok with dropping support for some of these formats. Some are pretty weird.